libs/react-spa/bff/src/lib/BffProvider.tsx

```diff
@@ -100,6 +134,13 @@ export const BffProvider = ({
         return
       }
 
+      // If user is logged in and on the old login path, then start the sign-in process
```

Comment:
I now this seams wrong, but the target link uri, prompt or login hint can have changed.